storage/mountscan/mountscan.go

```diff
@@ -0,0 +1,206 @@
+package mountscan
+
+import (
+	"io/ioutil"
+	"strconv"
+	"strings"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/contiv/errored"
+	"github.com/contiv/volplugin/errors"
+)
+
+const (
+	mountInfoFile           = "/proc/self/mountinfo"
+	deviceInfoFile          = "/proc/devices"
+	nfsMajorID              = 0
+	totalMountInfoFieldsNum = 11
+)
+
+// GetMountsRequest captures all the params required for scanning mountinfo
+type GetMountsRequest struct {
+	DriverName   string // ceph, nfs
+	FsType       string // nfs4, ext4
+	KernelDriver string // rbd, device-mapper, etc.
```

Comment:
I wanted this too be generic in `mountscan`. Also, this becomes handy when we have few more drivers down the line. Let me know your thoughts.